modules/fb_instant_articles_api/includes/admin.inc

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/**
+ * @file
+ * Settings for Facebook Instant Articles API module.
+ */
+
+/**
+ * Form constructor for Facebook Instant Articles API settings form.
+ */
+function fb_instant_articles_api_settings() {
+  $args = array(
+    '@development_mode' => 'https://developers.facebook.com/docs/instant-articles/api#create-fields',
+  );
+
+  // The Facebook app connection info could alternatively be stored via the fb
+  // module, but it's not yet ported to D8, and a bit cumbersome just for this.
+  $form['fb_instant_articles_api_app_id'] = array(
+    '#title' => t('Facebook App ID'),
+    '#type' => 'textfield',
+    '#default_value' => variable_get('fb_instant_articles_api_app_id'),
```

Comment:
Can we change this to `facebook_app_id` so if the application already using the facebook functionality then they don't need to add it again. Same for secret - `facebook_app_secret`